src/main/java/com/google/codeu/servlets/CountryServlet.java

```diff
@@ -0,0 +1,77 @@
+package com.google.codeu.servlets;
+
+import com.google.appengine.api.users.UserService;
+import com.google.appengine.api.users.UserServiceFactory;
+import com.google.codeu.data.Country;
+import com.google.codeu.data.Datastore;
+import com.google.codeu.data.Message;
+import java.io.IOException;
+import java.util.List;
+import java.util.Scanner;
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+@WebServlet("/country/*")
```

Comment:
Is this servlet also going to handle requests to subcategories for the country? For example, /country/Canada/Food ?

Reply:
I think it would be good to do so, eventually. Right now, it doesn't have separate processing for different urls, but I do think that would be a good way to set up that url and use wildcarding